src/svg-icons/index-generator.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+const rrs = require('recursive-readdir-sync');
+
+var outArray = [];
```

Comment:
`outArray` is a mutated object, `push` is not changing the reference, hence we should use a `const`. If we were using immutable push operation, I would agree, we would need a `let`.